files/pagerduty.rb

```diff
@@ -18,12 +18,18 @@ def api_key
   def trigger_incident
     return false unless api_key
     require 'redphone/pagerduty'
-    Redphone::Pagerduty.trigger_incident(
+    response = Redphone::Pagerduty.trigger_incident(
       :service_key  => api_key,
       :incident_key => incident_key,
-      :description  => description,
+      :description  => description(1024),
```

Comment:
https://developer.pagerduty.com/documentation/integration/events/trigger

So that is good